pkg/issuer/vault/setup.go

```diff
@@ -79,63 +79,63 @@ func (v *Vault) Setup(ctx context.Context) error {
 		return nil
 	}
 	if appRoleAuth != nil && len(appRoleAuth.SecretRef.Key) == 0 {
-		logf.V(logf.WarnLevel).Infof("%s: %s", v.issuer.GetObjectMeta().Name, messageAppRoleAuthKeyRequired)
+		logf.FromContext(ctx).V(logf.WarnLevel).Info(messageAppRoleAuthKeyRequired, "name", v.issuer.GetObjectMeta().Name)
 		apiutil.SetIssuerCondition(v.issuer, v.issuer.GetGeneration(), v1.IssuerConditionReady, cmmeta.ConditionFalse, errorVault, messageAppRoleAuthKeyRequired)
 		return nil
 	}
 
 	// When using the Kubernetes auth, giving a role is mandatory.
 	if kubeAuth != nil && len(kubeAuth.Role) == 0 {
-		logf.V(logf.WarnLevel).Infof("%s: %s", v.issuer.GetObjectMeta().Name, messageKubeAuthRoleRequired)
+		logf.FromContext(ctx).V(logf.WarnLevel).Info(messageKubeAuthRoleRequired, "name", v.issuer.GetObjectMeta().Name)
 		apiutil.SetIssuerCondition(v.issuer, v.issuer.GetGeneration(), v1.IssuerConditionReady, cmmeta.ConditionFalse, errorVault, messageKubeAuthRoleRequired)
 		return nil
 	}
 
 	// When using the Kubernetes auth, you must either set secretRef or
 	// serviceAccountRef.
 	if kubeAuth != nil && (kubeAuth.SecretRef.Name == "" && kubeAuth.ServiceAccountRef == nil) {
-		logf.V(logf.WarnLevel).Infof("%s: %s", v.issuer.GetObjectMeta().Name, messageKubeAuthEitherRequired)
+		logf.FromContext(ctx).V(logf.WarnLevel).Info(messageKubeAuthEitherRequired, "name", v.issuer.GetObjectMeta().Name)
 		apiutil.SetIssuerCondition(v.issuer, v.issuer.GetGeneration(), v1.IssuerConditionReady, cmmeta.ConditionFalse, errorVault, messageKubeAuthEitherRequired)
 		return nil
 	}
 
 	// When using the Kubernetes auth, you can't use secretRef and
 	// serviceAccountRef simultaneously.
 	if kubeAuth != nil && (kubeAuth.SecretRef.Name != "" && kubeAuth.ServiceAccountRef != nil) {
-		logf.V(logf.WarnLevel).Infof("%s: %s", v.issuer.GetObjectMeta().Name, messageKubeAuthSingleRequired)
+		logf.FromContext(ctx).V(logf.WarnLevel).Info(messageKubeAuthSingleRequired, "name", v.issuer.GetObjectMeta().Name)
 		apiutil.SetIssuerCondition(v.issuer, v.issuer.GetGeneration(), v1.IssuerConditionReady, cmmeta.ConditionFalse, errorVault, messageKubeAuthSingleRequired)
 		return nil
 	}
 
 	client, err := vaultinternal.New(ctx, v.resourceNamespace, v.createTokenFn, v.secretsLister, v.issuer)
 	if err != nil {
 		s := messageVaultClientInitFailed + err.Error()
-		logf.V(logf.WarnLevel).Infof("%s: %s", v.issuer.GetObjectMeta().Name, s)
+		logf.FromContext(ctx).V(logf.WarnLevel).Info(s, "name", v.issuer.GetObjectMeta().Name)
 		apiutil.SetIssuerCondition(v.issuer, v.issuer.GetGeneration(), v1.IssuerConditionReady, cmmeta.ConditionFalse, errorVault, s)
 		return err
 	}
 
 	if err := client.IsVaultInitializedAndUnsealed(); err != nil {
-		logf.V(logf.WarnLevel).Infof("%s: %s", v.issuer.GetObjectMeta().Name, err.Error())
+		logf.FromContext(ctx).V(logf.WarnLevel).Info(err.Error(), "name", v.issuer.GetObjectMeta().Name)
```

Comment:
```suggestion
		logf.FromContext(ctx).V(logf.WarnLevel).Error(err, "While checking if Vault is initialized and unsealed", "name", v.issuer.GetObjectMeta().Name)
```

And does it make sense to include the namespace here as well?